Snippets/Core/Core_8/Transports/QueueCreation/QueueBindings.cs

```diff
@@ -11,7 +11,7 @@ public class FeatureThatRequiresAQueue :
         protected override void Setup(FeatureConfigurationContext context)
         {
             var queueBindings = context.Settings.Get<QueueBindings>();
-            queueBindings.BindReceiving("someQueue");
+            queueBindings.BindSending("someQueue");
```

Comment:
looking at the docs text related to this, this is a weird documentation in the first place. Is this supposed to document how users can get NServiceBus to create additional queues?

Reply:
@timbussmann yes. This is my understanding.